packages/esm-service-queues-app/src/patient-info/patient-banner-queue-entry-status.scss

```diff
@@ -0,0 +1,29 @@
+@use '@carbon/colors';
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.queueEntryStatusContainer {
+  display: flex;
+  align-items: baseline;
```

Comment:
Now I've seen this in practice, I think this should be: 

```scss
	align-items: center;
  justify-items: center;
```

Reply:
FWIW:

Baseline:
![baseline](https://github.com/user-attachments/assets/eb615c6d-2ed8-419f-b613-c3dd155075fe)

Center:
![center](https://github.com/user-attachments/assets/2de8cc57-4c21-41c0-87b1-6fe1dbf40e37)

Center definitely looks better.

I don't think justify-items does anything here? (Though I do think there's probably too much padding on the "Change" button).